frontend/public/components/masthead-toolbar.jsx

```diff
@@ -174,6 +174,10 @@ class MastheadToolbar_ extends React.Component {
     window.open(window.SERVER_FLAGS.requestTokenURL, '_blank').opener = null;
   }
 
+  _getAdditionalLinks(links, type) {
```

Comment:
I'll doc in the API resource that they will be ordered alphabetically.

Reply:
Do we need to clarify? If we doc that, we might not be able to change it later

Reply:
Works, I'll not then.